### PR TITLE
Move web.config and use ANCM

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Review: @Tratcher 

RC2 Approval: @Eilon @DamianEdwards

### CODE REVIEW

File: samples/Builder.Filtering.Web/web.config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0"?>
+<configuration>
+  <system.webServer>
+    <handlers>
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
+    </handlers>
+    <aspNetCore forwardWindowsAuthToken="true" processPath="./Builder.Filtering.Web.exe" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout.log" />
```
Comment (Tratcher) [inline samples/Builder.Filtering.Web/web.config]: Why do you use a different parameter ordering in this repo?

File: samples/Localization.StarterWeb/project.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "Microsoft.AspNetCore.Diagnostics": "1.0.0-*",
     "Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore": "1.0.0-*",
     "Microsoft.AspNetCore.Identity.EntityFrameworkCore": "1.0.0-*",
-    "Microsoft.AspNetCore.IISPlatformHandler": "1.0.0-*",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0-*",
```
Comment (Tratcher) [inline samples/Localization.StarterWeb/project.json]: Sort

File: samples/Owin.Nowin.HelloWorld/project.json [hunk 3]
```diff
@@ -9,11 +9,7 @@
   "compilationOptions": {
```
Comment (Tratcher) [inline samples/Owin.Nowin.HelloWorld/project.json]: Add IISIntegration to these Nowin samples.

Comment (JunTaoLuo) [inline samples/Owin.Nowin.HelloWorld/project.json]: There seems to be some issues with websockets on iis but I'll merge this and then investigate.

**Pull request merged**